src/firefly/js/templates/fireflyviewer/LandingPage.jsx

```diff
@@ -0,0 +1,220 @@
+import {Button, Divider, Sheet, Snackbar, Stack, Typography} from '@mui/joy';
+import React, {useContext, useState} from 'react';
+import QueryStats from '@mui/icons-material/QueryStats';
+import TipsAndUpdates from '@mui/icons-material/TipsAndUpdates';
+import PropTypes from 'prop-types';
+
+import {getBackgroundInfo} from '../../core/background/BackgroundUtil.js';
+import {dispatchShowDropDown} from '../../core/LayoutCntlr.js';
+import {AppPropertiesCtx} from '../../ui/AppPropertiesCtx.jsx';
+import {useStoreConnector} from '../../ui/SimpleComponent.jsx';
+import {FileDropZone} from '../../visualize/ui/FileUploadViewPanel.jsx';
+import {dispatchAddPreference, getPreference} from 'firefly/core/AppDataCntlr';
+
+
+export function LandingPage({slots={}, slotProps={}}) {
+    const {appTitle,footer,
+        fileDropEventAction='FileUploadDropDownCmd'} = useContext(AppPropertiesCtx);
+
+    const {tabsMenuHint: tabsMenuHintProps, bgMonitorHint: bgMonitorHintProps, topSection: topSectionProps,
+        bottomSection: bottomSectionProps} = slotProps;
+
+    const {
+        tabsMenuHint=(
+            <AppHint appTitle={appTitle} id='tabsMenu'
+                     hintText='Choose a search from the tabs menu'
+                     {...tabsMenuHintProps}
+                     sx={{left: '16rem', ...tabsMenuHintProps?.sx}}/>
+        ),
+        bgMonitorHint=(
+            <AppHint appTitle={appTitle} id='bgMonitor'
+                     hintText='Load job results from background monitor'
+                     tipPlacement='end'
+                     {...bgMonitorHintProps}
+                     sx={{right: 8, ...bgMonitorHintProps?.sx}}/>
+        ),
+        topSection=(
+            <DefaultAppBranding appTitle={appTitle} {...topSectionProps}/>
+        ),
+        bottomSection=(
+            <EmptyResults {...bottomSectionProps}/>
+        )
+    } = slots;
```

Comment:
JoyUI implements `slots/slotProps` differently.  `slots` are of elementType, not Node.
Maybe, something like this.
```js
const Top = slots?.top || DefaultAppBranding;
const topProps = slots?.top ? {something} : defaultAppBrandingProps;

return (
    <Top {...topProps} {...slotProps?.top}/>
);

```
After reviewing your code, this may work better for us, since we may not need the extra flexibility.  But, I am a little concern about using `slots/slotProps`, yet implementing it differently.  If you would like to keep this, I suggest changing slots to `sections` or something other then `slots`.

Reply:
Just to make sure I understand it right: We still can't use elementType as string  (e.g. 'div') - how JoyUI does it. But we can mimic same behavior by passing React component aka function (`DefaultAppBranding`) instead of the Node (`<DefaultAppBrnading {...someProps}/>`)?